src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -44,7 +44,7 @@ def generate_prompt_tool(name, content, prompt_only=False, source=None):
                 "Generate tool meta failed for {tool_type} tool. Jinja parsing failed at line {line_number}: "
                 "{error_type_and_message}"
             ),
-            tool_type=tool_type,
+            tool_type=tool_type.value,
```

Comment:
test case covered?

Reply:
Yes, there is test case in https://github.com/microsoft/promptflow/pull/1285/files